### PR TITLE
docs: update "Other protocols" Plugins

### LINKED ISSUE #6734: docs: Update plugin documentation
### Current State

The plugin docs are inconsistent.

### Desired State

The plugin docs should be made consistent to enhance readability and make it easy to navigate.

It should also follow the documentation style guide: https://github.com/apache/apisix-website/pull/974

### PR DESCRIPTION
**Pull request by navendu-pottekkat**
Signed-off-by: Navendu Pottekkat <navendupottekkat@gmail.com>

### Description

Updates the documentation of the "Other protocols" Plugins. Fixes #6734 

Child PR of #6734

### CODE REVIEW

File: docs/en/latest/plugins/mqtt-proxy.md [hunk 1]
```diff
@@ -57,9 +61,9 @@ For example, the following configuration represents listening on the 9100 TCP po
     ...
 ```
 
-Then send the MQTT request to port 9100.
+You can now send the MQT request to port `9100`.
```
Comment (juzhiyuan) [inline docs/en/latest/plugins/mqtt-proxy.md:64]:
```suggestion
You can now send the MQTT request to port `9100`.
```

**Review by membphis — COMMENTED** [review]
LGTM

File: docs/en/latest/plugins/dubbo-proxy.md [hunk 2]
```diff
@@ -23,40 +30,39 @@ title: dubbo-proxy
...
 ## Runtime Attributes
 
-| Name         | Type   | Requirement | Default  | Valid        | Description                                                          |
-| ------------ | ------ | ----------- | -------- | ------------ | -------------------------------------------------------------------- |
-| service_name    | string | required    |          |              | dubbo provider service name|
-| service_version | string | required    |          |              | dubbo provider service version|
-| method          | string | optional    | the path of uri   |     | dubbo provider service method|
+| Name            | Type   | Required | Default              | Description                     |
+| --------------- | ------ | -------- | -------------------- | ------------------------------- |
+| service_name    | string | required |                      | Dubbo provider service name.    |
```
Comment (okaybase) [inline docs/en/latest/plugins/dubbo-proxy.md:45]:
```suggestion
| service_name    | string | True |                      | Dubbo provider service name.    |
```

File: docs/en/latest/plugins/dubbo-proxy.md [hunk 3]
```diff
@@ -23,40 +30,39 @@ title: dubbo-proxy
...
+| service_version | string | required |                      | Dubbo provider service version. |
```
Comment (okaybase) [inline docs/en/latest/plugins/dubbo-proxy.md:46]:
```suggestion
| service_version | string | True |                      | Dubbo provider service version. |
```

File: docs/en/latest/plugins/dubbo-proxy.md [hunk 4]
```diff
@@ -23,40 +30,39 @@ title: dubbo-proxy
...
+| method          | string | optional | The path of the URI. | Dubbo provider service method.  |
```
Comment (okaybase) [inline docs/en/latest/plugins/dubbo-proxy.md:47]:
```suggestion
| method          | string | False | The path of the URI. | Dubbo provider service method.  |
```

File: docs/en/latest/plugins/dubbo-proxy.md [hunk 5]
```diff
@@ -23,40 +30,39 @@ title: dubbo-proxy
...
 
 ## Static Attributes
 
-| Name         | Type   | Requirement | Default  | Valid        | Description                                                          |
-| ------------ | ------ | ----------- | -------- | ------------ | -------------------------------------------------------------------- |
-| upstream_multiplex_count | number | required    | 32        | >= 1 | the maximum number of multiplex requests in an upstream connection |
+| Name                     | Type   | Required | Default | Valid values | Description                                                     |
+| ------------------------ | ------ | -------- | ------- | ------------ | --------------------------------------------------------------- |
+| upstream_multiplex_count | number | required | 32      | >= 1         | Maximum number of multiplex requests in an upstream connection. |
```
Comment (okaybase) [inline docs/en/latest/plugins/dubbo-proxy.md:53]:
```suggestion
| upstream_multiplex_count | number | True | 32      | >= 1         | Maximum number of multiplex requests in an upstream connection. |
```

File: docs/en/latest/plugins/dubbo-proxy.md [hunk 6]
```diff
@@ -23,40 +30,39 @@ title: dubbo-proxy
 
 ## Description
 
-dubbo-proxy plugin allows you proxy HTTP request to [**dubbo**](http://dubbo.apache.org).
+The `dubbo-proxy` Plugin allows you to proxy HTTP requests to [Apache Dubbo](https://dubbo.apache.org/en/index.html).
+
+:::info IMPORTANT
 
-## Requirement
+If you are using OpenResty, you need to build it with Dubbo support. See [How do I build the APISIX base environment](./../FAQ.md#how-do-i-build-the-apisix-base-environment) for details.
```
Comment (hf400159) [inline docs/en/latest/plugins/dubbo-proxy.md:37]:
```suggestion
If you are using OpenResty, you need to build it with Dubbo support. See [How do I build the APISIX base environment](./../FAQ.md#how-do-i-build-the-apisix-base-environment?) for details.
```

Comment (navendu-pottekkat) [reply docs/en/latest/plugins/dubbo-proxy.md:37]: @hf400159 The "?" does not come in the URL it seems... See: https://apisix.apache.org/docs/apisix/FAQ#how-do-i-build-the-apisix-base-environment

**Comment by navendu-pottekkat** [comment]
Thank you @okaybase and @hf400159

I have made the changes as suggested.

**Review by hf400159 — APPROVED** [review]

**Review by SylviaBABY — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**